app/controllers/check_ins_controller.rb

```diff
@@ -42,14 +43,14 @@ def update
   end
 
   private def merge_worked_at_and_account_into_log_entries(check_in_params)
-    check_in_params[:log_entries_attributes].each do |_id, log_entry|
+    check_in_params.fetch(:log_entries_attributes, {}).each do |_id, log_entry|
```

Comment:
More importantly, we're now making  sure that if the check_in_params doesn't have log_entries_attributes  inside of it, we get something that won't explode out of the params (in this case an empty hash)